kolibri_instant_schools_plugin/assets/src/views/about/views/about/assets/style.css

```diff
@@ -18,7 +18,7 @@ h3, h4, h5, h6 {
   font-size: 1em;
 }
 
-.vodafone-logo, .instant-schools-logo {
+.vodafone-logo {
```

Comment:
Can you also remove the other css for .instant-schools-logo

Reply:
done